DuckDuckGo/SafariSearchInstructionsViewController.swift

```diff
@@ -70,8 +88,12 @@ class SafariSearchInstructionsViewController: UIViewController {
         let minimumMargin = SafariSearchInstructionsViewController.minimumTopMargin
         topMarginConstraint.constant = marginForVerticalCentering > minimumMargin ? marginForVerticalCentering : minimumMargin
     }
-    
-    @IBAction func onDonePressed(_ sender: UIButton) {
-        dismiss(animated: true, completion: nil)
+
+    private func makeStyle() -> Style {
+        return Style("highlight", {
+            $0.color = UIColor(hex: "#333333", alpha: 1.0)
+        })
```

Comment:
Agreed - I prefer this style myself. Probably because I copied and changed the example from the docs.